keras_hub/src/models/efficientnet/efficientnet_backbone.py

```diff
@@ -99,8 +106,8 @@ class EfficientNetBackbone(FeaturePyramidBackbone):
     def __init__(
         self,
         *,
-        width_coefficient,
-        depth_coefficient,
+        stackwise_width_coefficient=None,
```

Comment:
probably make these plural, in keeping with other args? stackwise_width_coefficients and stackwise_depth_coefficients

Reply:
done